packages/cli/src/interfaces/jwt.interface.ts

```diff
@@ -0,0 +1,13 @@
+export interface IJwt {
```

Comment:
@djabarovgeorge Do we have IJWTPayload in notifire shared? If no, I suggest adding it there and refactoring the API to import it from there instead aswell. So we won't have duplications

Reply:
I did not see one that why I created one, it is a good idea to export it to the notifire shared I will look into it.